JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt

```diff
@@ -62,7 +62,7 @@ fun PostImage(modifier: Modifier = Modifier.None, post: Post) {
     val image = post.imageThumb ?: imageResource(R.drawable.placeholder_1_1)
 
     Container(modifier = modifier + LayoutSize(40.dp, 40.dp)) {
-        DrawImage(image)
+        SimpleImage(image =  image)
```

Comment:
Ditto

Reply:
We need this one for the sizing / padding. `SimpleImage` doesn't take a modifier.

Reply:
Same, swapped for ImagePainter.